src/coreclr/nativeaot/System.Private.CoreLib/src/System/MulticastDelegate.cs

```diff
@@ -1,11 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.ComponentModel;
 using System.Diagnostics;
 using System.Diagnostics.CodeAnalysis;
-using System.Reflection;
-using System.Runtime;
 using System.Runtime.CompilerServices;
 using System.Runtime.Serialization;
 
```

Comment:
If the remaining instance methods are moved too, we may be able to mark `System.Delegate fields as `private`.